src/core/stdcpp/string.d

```diff
@@ -0,0 +1,204 @@
+﻿/**
+ * D header file for interaction with C++ std::string.
+ *
+ * Copyright: Copyright Guillaume Chatelet 2014 - 2015.
```

Comment:
If possible maybe assign the copyright to the D Language Foundation.
CC @gchatelet

Reply:
Sure, I respected the style that was in place back then but feel free to change it to whatever is needed.